src/components/environment-badge.tsx

```diff
@@ -36,9 +37,13 @@ export function EnvironmentBadge( { type, selected }: EnvironmentBadgeProps ) {
 		production: __( 'Production' ),
 	};
 
-	return <Badge className={ cx( getClassName() ) }>{ labels[ type ] }</Badge>;
+	return <Badge className={ cx( getClassName(), className ) }>{ labels[ type ] }</Badge>;
 }
 
+export const StudioBadge = ( { className }: { className?: string } ) => {
+	return <Badge className={ cx( 'bg-a8c-gray-5 text-a8c-gray-80', className ) }>Studio</Badge>;
```

Comment:
Maybe we should translate Studio too.
```suggestion
	return <Badge className={ cx( 'bg-a8c-gray-5 text-a8c-gray-80', className ) }>{ __( 'Studio' ) }</Badge>;
```